### ISSUE TITLE
"Workspace: Add and Commit" giving 'error running command: The "path" argument must be of type string. Received an instance of Object'

### ISSUE THREAD

**Issue by djradon** [dendronhq/dendron#1297]
## Please select the area your Bug applys to. (Multiple selections are Possible)

- [ ] Onboard. Initial vault setup and import/export pods 
- [ ] Create. Note creation, lookup, snippets and templates
- [ ] Retrieve. Backlinks, references, graph view
- [ ] Structure. Refactoring, multi-vault and schemas
- [ ] Publish. Sharing your repo with the world
- [x] Misc (Choose this if your not sure)

## Describe the bug

">Dendron: Workspace: Add and Commit" is giving me and @kevins8  'error running command: The "path" argument must be of type string. Received an instance of Object' -- that's with both my pre-existing multi-vault workspace and a newly-initialized test vault.

## Desktop (please complete the following information)

- Windows
- VS Code
- Dendron v0.57.0

**Comment by viddo** [comment]
I am getting this same error every time I run the command. 

Checking repo status in terminal I can a commit was properly done and everything looks legit as far as I can tell. It's not a blocker but still annoying.

@kevinslin re: 🏷️ `status.triage-needed`, considering it's consistent for me is there some additional info I (or the prev reporters) can provide to help troubleshoot and find the culprit?

FWIW my stack trace appears to look somewhat same/similar to @matthew-green-nandos  above? 
Copy paste of the relevant line in log file (with some redactions and formatting to make it easier to read):

```JS
{
  level: 50,
  time: 1642421066507,
  pid: "█████",
  hostname: "████.home",
  name: "dendron",
  ctx: "dendron.addAndCommit:run",
  error: {
    stack:
      'TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received an instance of Object' +
      "at validateString (internal/validators.js:124:11)" +
      "at Object.basename (path.js:1156:5)" +
      "at /████/████/.vscode-insiders/extensions/dendron.dendron-0.77.1/dist/extension.js:8:3069431" +
      "at It (/████/████/.vscode-insiders/extensions/dendron.dendron-0.77.1/dist/extension.js:8:1284565)" +
      "at Function.ko [as map] (/████/████/.vscode-insiders/extensions/dendron.dendron-0.77.1/dist/extension.js:8:1319456)" +
      "at Object.execute (/████/████/.vscode-insiders/extensions/dendron.dendron-0.77.1/dist/extension.js:8:3069417)" +
      "at runMicrotasks (<anonymous>)" +
      "at processTicksAndRejections (internal/process/task_queues.js:93:5)" +
      "at Object.run (/████/████/.vscode-insiders/extensions/dendron.dendron-0.77.1/dist/extension.js:8:2883795)" +
      "at /████/████/.vscode-insiders/extensions/dendron.dendron-0.77.1/dist/extension.js:8:3166261" +
      "at m._executeContributedCommand (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:95:31884)",
    message:
      'error while running command: The "path" argument must be of type string. Received an instance of Object',
    name: "DendronError",
    status: "unknown",
    payload: "{}",
    innerError: { code: "ERR_INVALID_ARG_TYPE" },
  },
}
```

**Comment by Trerot** [comment]
error while running command: The "path" argument must be of type string. Received an instance of Object

If there is anything, just ask.



```json
{
    "level": 50,
    "time": 1642498487170,
    "pid": "pid",
    "hostname": "hostname",
    "name": "dendron",
    "ctx": "dendron.addAndCommit:run",
    "error": {
        "stack": "TypeError [ERR_INVALID_ARG_TYPE]: The \"path\" argument must be of type string. Received an instance of Object
                at validateString (internal/validators.js:124:11)
                at Object.basename (path.js:671:5)
                at c:\\Users\\USERNAME\\.vscode\\extensions\\dendron.dendron-0.77.1\\dist\\extension.js:8:3069431
                at It (c:\\Users\\USERNAME\\.vscode\\extensions\\dendron.dendron-0.77.1\\dist\\extension.js:8:1284565)
                at Function.ko [as map] (c:\\Users\\USERNAME\\.vscode\\extensions\\dendron.dendron-0.77.1\\dist\\extension.js:8:1319456)
            at Object.execute (c:\\Users\\USERNAME\\.vscode\\extensions\\dendron.dendron-0.77.1\\dist\\extension.js:8:3069417)
            at async Object.run (c:\\Users\\USERNAME\\.vscode\\extensions\\dendron.dendron-0.77.1\\dist\\extension.js:8:2883795)
            at async c:\\Users\\USERNAME\\.vscode\\extensions\\dendron.dendron-0.77.1\\dist\\extension.js:8:3166261
            at async r._executeContributedCommand (c:\\Users\\USERNAME\\AppData\\Local\\Programs\\Microsoft VS Code\\resources\\app\\out\\vs\\workbench\\services\\extensions\
        ode\\extensionHostProcess.js:95:31072)",
        "message": "error while running command: The \"path\" argument must be of type string. Received an instance of Object",
        "name": "DendronError",
        "status": "unknown",
        "payload": "{}",
        "innerError": {
            "code": "ERR_INVALID_ARG_TYPE"
        }
    }
}
```

## Desktop
* Windows. 3 different machines, 2x win 10, 1x win 11.
* VS Code
* Dendron v0.77.1

**Comment by Harshita-mindfire** [comment]
This should be fixed in v0.87 of Dendron. Feel free to comment here if not the same.
